integration/spark2/src/test/scala/org/apache/spark/carbondata/restructure/AlterTableValidationTestCase.scala

```diff
@@ -775,13 +775,12 @@ test("test alter command for boolean data type with correct default measure valu
       .addProperty(CarbonCommonConstants.CARBON_TIMESTAMP_FORMAT, timestampFormat)
   }
 
-
   test("Alter Table Change Sort Scope 1") {
     sql("DROP TABLE IF EXISTS t1")
     sql(s"CREATE TABLE t1(age int, name string) STORED BY 'carbondata' TBLPROPERTIES" +
         s"('sort_columns'='age', 'sort_scope'='local_sort')")
     sql("ALTER TABLE t1 SET TBLPROPERTIES('sort_scope'='batch_sort')")
-    assert(sortScopeInDescFormatted("t1").equalsIgnoreCase("BATCH_SORT"))
+    assert(sortScopeInDescFormatted("t1").equalsIgnoreCase("Some(BATCH_SORT)"))
```

Comment:
instead of changing the test case can you change describe formatted command to show only the scope and not the Option. Currently toString is called on Option therefore it is showing Some("batch_sort")